apps/website-new/docs/zh/practice/bridge/react-bridge.mdx

```diff
@@ -0,0 +1,217 @@
+# React Bridge（for React v16、v17、v18）
+
+`@module-federation/bridge-react` 提供了用于 React 应用的 `bridge` 工具函数，其提供的 `createBridgeComponent` 可用于导出应用级别模块，`createRemoteComponent` 用于加载应用级别模块。[Demo](https://github.com/module-federation/core/blob/6756c9142d0bf41262a26fe30e3c39d7957cfa58/apps/router-demo)
+
+
+
+### 安装
+
+import { PackageManagerTabs } from '@theme';
+
+<PackageManagerTabs
+  command={{
+    npm: 'npm install @module-federation/bridge-react@0.0.0-next-20240620060503',
+    yarn: 'yarn add @module-federation/bridge-react@0.0.0-next-20240620060503',
+    pnpm: 'pnpm add @module-federation/bridge-react@0.0.0-next-20240620060503',
+  }}
+/>
+
+
+### 示例
+
+> Remote
+
+```tsx
+// ./src/export-app.tsx
+import App from './src/App.tsx';
+import { createBridgeComponent } from '@module-federation/bridge-react';
+
+export default createBridgeComponent({
+  rootComponent: App
+});
+```
+
+```ts
+// rsbuild.config.ts
+export default defineConfig({
+  source: {
+    alias: {
+      // 设置 react-router-dom 到 proxy 的 router 中，方便
+      'react-router-dom$': path.resolve(
```

Comment:
can we patch this config in webpack/rspack plugin ?

Reply:
What is the best way to add this by default when the conditions are met? I am worried that this will affect